src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -941,6 +930,19 @@
       <Link>Interop\Windows\BCrypt\Interop.BCryptGenRandom.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="('$(TargetsWindows)'=='true')  And '$(EnableDummyGlobalizationImplementation)' != 'true'">
```

Comment:
Nit: `('$(TargetsWindows)'=='true')` -> `'$(TargetsWindows)'=='true'` to match the style used in other places